app/views/admin/appointments/_appointment.html.erb

```diff
@@ -0,0 +1,251 @@
+<div class="card mt-3 mb-4">
+  <div class="card-header">
+    <h5 class="card-title">
+      <% if appointment.dose_sequence_number.present? -%>
+        <%= appointment.dose_sequence_number %><sup>a</sup> dose
+        <% if appointment.dose_vaccine.present? -%>
+          da <%= appointment.dose_vaccine&.name %>
+        <% end -%> &mdash; <%= t(:"appointments.state.#{@appointment.state}") %>
+      <% else -%>
+        Agendamento vago
+      <% end -%>
+    </h5>
+  </div>
+  <div class="card-body">
+    <table class="table">
+      <tbody>
+      <tr>
+        <th scope="row">Paciente:</th>
+        <td>
+          <% if appointment.patient_id.present? -%>
+            <p>
+              <%= link_to appointment.patient.name, [:admin, appointment.patient] %>
+            </p>
+            <p>
+              <strong>CPF:</strong> <%= ApplicationHelper.humanize_cpf(appointment.patient.cpf) %>
+              <br/>
+              <strong>Data de Nascimento:</strong> <%= l(appointment.patient.birth_date.to_date) %>
+              <br/>
+              <strong>Nome da mãe:</strong> <%= appointment.patient.mother_name %>
+              <br/>
+              <strong>Bairro:</strong> <%= appointment.patient.neighborhood.name %>
+              <br/>
+              <strong>Telefones:</strong> <%= appointment.patient.phones %>
+            </p>
+            <p>
+              <strong>Grupos que pertence:</strong>
+              <% if appointment.patient.groups.any? -%>
+                <ul>
+                  <% appointment.patient.groups.admin_order.each do |group| -%>
+                    <li>
+                      <%= group.name_with_parent %>
+                    </li>
+                  <% end -%>
+                </ul>
+              <% else -%>
+                Não preencheu nenhum grupo.
+              <% end -%>
+            </p>
+
+            <% if appointment.can_remove_patient? -%>
```

Comment:
Não sei se essa regra de "negócio" chegou a ser discutida, mas testando verifiquei que operadores que **não são Administradores** de uma unidade específica podem pesquisar por qualquer paciente e executar as ações de:
- Cancelar Agendamento;
- Realizar e reveerter Check-in / check-out;
- Suspender agendamentos;

de agendamentos que não foram realizadas na unidade deste operador. Isso me deixou um pouco preocupado com uma possível interferência de operadores de uma unidade em agendamentos de outras unidades.

Talvez seja interessante colocar mais restrições para essas ações, para ser feita só por Administradores ou Operadores da mesma unidade do agendamento, por exemplo:

```suggestion
            <% if appointment.can_remove_patient? && 
            (current_user.ubs.pluck(:name).include? appointment.ubs.name || current_user.admin? -%>
```